### ISSUE TITLE
Some weather files don't work

### ISSUE THREAD

**Issue by amraladdin** [architecture-building-systems/CEAforArcGIS#241]
I used the following weather files while simulating... Data helper works fine, however, when running the demand, only the first 5 buildings run, while the rest of the buildings don't, and I have to force the software to shut down. The files used are the following (only EPW):
https://energyplus.net/weather-location/africa_wmo_region_1/EGY//EGY_Cairo.623660_IWEC
https://energyplus.net/weather-location/africa_wmo_region_1/EGY//EGY_Cairo.Intl.Airport.623660_ETMY
https://energyplus.net/weather-location/europe_wmo_region_6/SWE//SWE_Kiruna.020440_IWEC

With the last weather file, it ran 267 buildings and then stopped.
I checked each case twice.

**Comment by amraladdin** [comment]
@JIMENOFONSECA Any idea who can take a look at this point?

**Comment by JIMENOFONSECA** [comment]
I can take a look, create a branch and upload the files to the weather folder. Before i do something. did you run the radiation file first? and then the demand?

**Comment by amraladdin** [comment]
No, I didn't run the radiation files, only the properties and then demand directly. However, it worked without running the radiation file for Singapore, Sri Lanka, Milano and Palermo...

**Comment by JIMENOFONSECA** [comment]
Could you run it all and let me know the results?

Best, Jimeno

On 19 Jul 2016, at 20:42, Amr Elesawy <notifications@github.com<mailto:notifications@github.com>> wrote:


No, I didn't run the radiation files, only the properties and then demand directly. However, it worked without running the radiation file for Singapore, Sri Lanka, Milano and Palermo...

**Comment by amraladdin** [comment]
Yes, I'll do that and update.

**Comment by JIMENOFONSECA** [comment]
#241 and #234 can you check if it is working @amraladdin

**Comment by amraladdin** [comment]
@JIMENOFONSECA Yes, I'm on it... Just need to install the latest ArcGIS and CEA

**Comment by amraladdin** [comment]
Issue solved when I run the radiation file for Cairo. Checking Kiruna (Swecden) now...

**Comment by amraladdin** [comment]
So, when I try to run the radiation for Kiruna (Sweden), I get the following error:

![radiation_error_kiruna](https://cloud.githubusercontent.com/assets/16256867/17863795/88ce071a-689b-11e6-857f-727b9744153a.jpg)

Any idea what this might be? Or how to solve it?

In any case, this is not important for the MIBS students, so I moved the Milestone to 1.8, still, I don't think we would want to keep this issue standing.

**Comment by martin-mosteiro** [comment]
@amraladdin I believe #341 had quite a bit in common with the errors we were getting for Cairo. So you might want to try these weather files again and see if they work now.

**Comment by amraladdin** [comment]
UPDATE:
----------

I ran the radiation file for Cairo (check .epw file above) and this time it worked! I ran the data helper and demand as well and they worked fine as well.

However, when I tried to run the radiation file for Kiruna Climate in Sweden, I still get the following error:

![kiruna error](https://cloud.githubusercontent.com/assets/16256867/20796635/50b7d82a-b7d7-11e6-8432-caa70122856a.jpg)

Does this mean there might be something wrong with the labeling? @JIMENOFONSECA @daren-thomas Any idea how to fix this?

**Comment by JIMENOFONSECA** [comment]
I see the problem. we are running kiruna weather file with the location of Zug/Zurich. The latitute and longitud of the site should match the weather file in order of this to work. In order words. please use the latest version of the radiation tool which requires you to put the latitude and longitud of the location where you are doing the calculations.

**Comment by amraladdin** [comment]
Sounds good! Thanks!
I'll close this issue then...

**Comment by daren-thomas** [comment]
sounds good is not enough: did jimenos proposal work? did you test it?

**Comment by amraladdin** [comment]
You are absolutely right. I apologize, I understood from Jimeno's message that they ran the simulation for the reference cases of Zurich or Zug, while I guess he meant that this is the cause of the problem. My mistake!

I used the latest radiation model. I checked from the Kiruna weather file the latitude and longitude values (67.82 and 20.33 respectively) However, I got another error. Could you please check again @JIMENOFONSECA ? 

Inputs:
![kiruna radiation input](https://cloud.githubusercontent.com/assets/16256867/20922690/4506189a-bba9-11e6-8c8e-b93d604cee12.jpg)

Erorr message:
AlwaysUpError: 'Sun' is still above the horizon at 2015/5/27 22:35:51

Error Screen Shot:
![kiruna error 02](https://cloud.githubusercontent.com/assets/16256867/20922670/38850f22-bba9-11e6-90f5-58c22e7a0381.jpg)

Weather File used:
[Kiruna.zip](https://github.com/architecture-building-systems/CEAforArcGIS/files/633587/Kiruna.zip)

**Comment by amraladdin** [comment]
@JIMENOFONSECA did you manage to take a look at this issue?

**Comment by JIMENOFONSECA** [comment]
@amraladdin yes you shoudl go to the calculation of the sunrise hours in the area... it is making problems and would need to be replaced.

**Comment by daren-thomas** [comment]
pyephem and sunrise has weird behaviour...

**Comment by amraladdin** [comment]
@daren-thomas @JIMENOFONSECA Do we have any updates on this issue? I'm not sure I'm able to fix this issue in the tool. Could you recommend someone to do so?

**Comment by JIMENOFONSECA** [comment]
Dis you try again if the problem still exists?

Best, Jimeno

On 21 Mar 2017, at 00:51, Amr Elesawy <notifications@github.com<mailto:notifications@github.com>> wrote:


@daren-thomas<https://github.com/daren-thomas> @JIMENOFONSECA<https://github.com/JIMENOFONSECA> Do we have any updates on this issue? I'm not sure I'm able to fix this issue in the tool. Could you recommend someone to do so?

**Comment by amraladdin** [comment]
@JIMENOFONSECA yes, and it gives the same error. @martin-mosteiro also tried running it and it generated the same error.

This opens an important point, I believe this issue is not a priority at this point. We don't have any research projects in such climates, however, if the tool is to be open to the public in the future, this issue might be of importance. That being said, I think we should go in 1 of the following directions:
- Someone from the developing team can take a look at this issue and fix it. 
- We delay this issue and give it a low priority.
- We close the issue.

I go for closing the issue and saving this point in a "someday/maybe" depository.

What do you guys think?

**Comment by martin-mosteiro** [comment]
Agreed, since it's an issue that won't affect any of our current case studies, it doesn't make sense to work on it. I say keep it as Priority 3 (and work on it iff we get such a case study) or (even better) close it.

**Comment by JIMENOFONSECA** [comment]
Actually, the know what is the cause.

You cannot run buildings and terrain located in a coordinate system away from the original climate.

To make it work in cairo. You would need buildings and terrain geolocated in there.

Best, Jimeno

On 21 Mar 2017, at 22:09, martin-mosteiro <notifications@github.com<mailto:notifications@github.com>> wrote:


Agreed, since it's an issue that won't affect any of our current case studies, it doesn't make sense to work on it. I say keep it as Priority 3 (and work on it iff we get such a case study) or (even better) close it.

**Comment by amraladdin** [comment]
I'm not sure if this is the case, cause the Cairo case ran successfully before (Check my update on the 1st of December 2016). So perhaps it is related to the sun hours availability. Any other possibilities?

**Comment by JIMENOFONSECA** [comment]
There is a chance the PR of daysim solar radiation fixes al this. so i would close this issue
________________________________
